train/arch-and-scaling-template.slurm

```diff
@@ -177,5 +177,13 @@ export CMD=" \
 
 echo $CMD
 
+# We create the folder where the logs will be stored. This command assumes that only the last folder of the logs path
+# can be missing
+if [[ ! -d "${LOGS_PATH}" ]]
+then
+    echo "${LOGS_PATH} doesn't exists on your filesystem. We create it."
+    mkdir $LOGS_PATH
+fi
```

Comment:
```suggestion
mkdir -p $LOGS_PATH
```

this is much simpler - and no need to announce this, as it's obvious.

Reply:
yes, indeed, when we had a separate repo it was just automatically taken care of.

In the tr8 setup I'm doing:
```
cd $six_ALL_CCFRSCRATCH//checkpoints/tr8-104B
mkdir checkpoints
git clone https://huggingface.co/bigscience/tr8-104B-logs
cd tr8-104B-logs
mkdir tensorboard codecarbon logs
transformers-cli lfs-enable-largefiles .
```

so all log dirs are there from the get going

Reply:
Thank you so much for your review @stas00 ! I agree with everything you say, your proposals are indeed more appropriate. We don't use codecarbon for our experiments yet, but do you think we should add `mkdir -p $CODECARBON_PATH` too ?

Reply:
it's totally your call - perhaps add it but commented out if you're not using it at the moment

Reply:
Great thanks! I added it :)